kount/client.py

```diff
@@ -79,7 +79,12 @@ def process(self, params):
                 logger.debug(error)
                 raise ValueError(error)
         else:
-            logger.debug("process json: %s", req_json)
+            roundtrip = request.elapsed.total_seconds()
+            merc = headers_api.get("X-Kount-Merc-Id", None)
```

Comment:
merc = headers_api.get("X-Kount-Merc-Id", None)
is just for logging purpose /in logger MERC = None if missing header "X-Kount-Merc-Id" /

but i'll remove it because it's already logged in row 63
```
logger.debug("url %s, headers %s, params %s", self.url,
                     headers_api, params)
```

the actual value is set in:

        try:
            headers_api["X-Kount-Merc-Id"] = params['MERC']
        except KeyError:
            message = "Required field 'MERC' is missing. \
                       Header's param 'X-Kount-Merc-Id' is not set."
            logger.debug(message)